template/router/haproxy.cfg.tmpl

```diff
@@ -72,6 +74,12 @@ backend project-creator
         {{range service "project-creator"}}
         server {{.Node}} {{.Address}}:{{.Port}} check cookie {{.Node}}_{{.Port}} {{end}}
 
+backend s3-club-7
+        http-request set-path /
+        balance leastconn
+        option httpclose
```

Comment:
Its a dumb proxy with a no real state- don't need to analyse every request in a connection. Its a lot more performant.